src/screens/ContactScreen/elements/Search.js

```diff
@@ -19,6 +19,7 @@ const Search = ({onPress, animatedValue, onChangeText, text, onClearText, isLoad
       <TextInput
         value={text}
         placeholder={'Search Users'}
+        placeholderTextColor={COLORS.gray400}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Adding a placeholder text color improves the user interface by providing better visibility. Good addition!
```diff
+ placeholderTextColor={COLORS.gray400}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->